dash/frontend/src/app/modules/private/pages/cluster/cluster-summary/cluster-summary.component.ts

```diff
@@ -96,21 +96,23 @@ export class ClusterSummaryComponent implements OnInit, AfterViewInit, OnDestroy
   page = 0;
   limit = 10;
   clusterEvents: IClusterEvent[];
-  expandStatus: boolean;
-  expandSubscription: Subscription;
   resizeTimeout;
+  logLoaderName = 'cluster-logs-loader';
+  logsLoaderConfig: NgxUiLoaderConfig = {
+    bgsType: SPINNER.rectangleBounce,
+    bgsPosition: POSITION.centerCenter,
```

Comment:
Were you able to validate that this positions it where you want it? (it looks like it'll put it in the center of the screen)

Reply:
Yes, the position is relative to its parent container, not the page. Here is a screenshot of where it is. (It is animated See rectangle-bounce here if you want to see that: https://tdev.app/ngx-ui-loader/demo/spinners)


<img width="1022" alt="Screenshot 2023-08-24 at 4 52 36 PM" src="https://github.com/m9sweeper/m9sweeper/assets/81321170/e6c65a74-0d54-4ed7-8611-1bcc961dfb42">

Reply:
Great!
I found with that loader that it often positioned it absolutely and it could be a pain to get it to stop.